devimint/src/main.rs

```diff
@@ -1114,59 +1114,93 @@ async fn setup(arg: CommonArgs) -> Result<(ProcessManager, TaskGroup)> {
     write_overwrite_async(globals.FM_TEST_DIR.join("env"), env_string).await?;
     info!("Test setup in {:?}", globals.FM_DATA_DIR);
     let process_mgr = ProcessManager::new(globals);
-    let task_group = TaskGroup::new();
-    task_group.install_kill_handler();
-    Ok((process_mgr, task_group))
+    Ok(process_mgr)
 }
 
 #[tokio::main]
 async fn main() -> Result<()> {
     let args = Args::parse();
     match args.command {
         Cmd::ExternalDaemons => {
-            let (process_mgr, task_group) = setup(args.common).await?;
+            let process_mgr = setup(args.common).await?;
+            let task_group = TaskGroup::new();
```

Comment:
@douglaz, I'm trying to understand the implication of this refactor for creating TG and installing kill handler. Does moving these two lines from within setup have a specific effect in killing processes later?

Reply:
oh maybe it's because tg is only used in `DevFed` and `ExternalDaemons` scenario?

Reply:
Yes, it's not used everywhere and I think being explicit is better if you are reading the code for the first time